yb-voyager/cmd/constants.go

```diff
@@ -32,6 +32,8 @@ const (
 	POSTGRESQL                  = "postgresql"
 	LAST_SPLIT_NUM              = 0
 	SPLIT_INFO_PATTERN          = "[0-9]*.[0-9]*.[0-9]*.[0-9]*"
+	COPY_AUTORETRY_COUNT        = 5
```

Comment:
Drop the word "AUTO" from everywhere.

Just COPY_MAX_RETRY_COUNT is sufficient.

Reply:
done